frontend/src/authentication/contexts/GenericAuthContext.js

```diff
@@ -279,6 +282,10 @@ export const GenericAuthProvider = (props) => {
         }
       });
     }
+
+    if (sessionStorage.getItem('window-location')) {
```

Comment:
According to [specification](https://developer.mozilla.org/en-US/docs/Web/API/Storage/removeItem), `removeItem` is safe, you don't have to check, if the item exists.